### ISSUE TITLE
Running Mean/Var for batchnorm not included in pretrained weights

### ISSUE THREAD

**Issue by youssef-osaro** [facebookresearch/Detectron#189]
The pretrained models on the model zoo seem to only contain the batchnorm scale and bias, but not the running means and variances. Should these be also included in the weight file?

**Comment by ppwwyyxx** [comment]
Mean and variance were absorbed into scale and bias to form a single affine layer.

**Comment by rbgirshick** [comment]
See https://github.com/facebookresearch/Detectron/blob/master/tools/pickle_caffe_blobs.py#L113.

**Comment by bhmobil** [comment]
Future's aimless
